scripts/helper/schedule-webworker.js

```diff
@@ -39,9 +39,70 @@ function getEndTime(date, startTime, endTime, totalNumber, number) {
 
 self.addEventListener('message', function (e) {
   var speakers = e.data.speakers;
+  var instructors = e.data.instructors;
   var sessions = e.data.sessions;
+  var workshops = e.data.workshops;
   var schedule = e.data.schedule;
 
+  for (var i = 0, instructorsLen = instructors.length; i < instructorsLen; i++) {
+    instructors[i].workshop = new Object()
+    instructors[i].workshop.price = "2990 NOK + VAT"
+    var workshopId = instructors[i].workshopId
+    if (workshops != null) {
+      for (var j = 0, workshopsLen = workshops.length; j < workshopsLen; j++) {
+        if (workshopId == workshops[j].id) {
+          instructors[i].workshop = workshops[j]
+          break
+        }
+      }
+    } 
+
+    // test
```

Comment:
While I couldn't get data from Firebase, I've hardcoded it and will fix it later this week